src/main/scala/ztis/relationships/RelationshipFetcherActor.scala

```diff
@@ -0,0 +1,77 @@
+package ztis.relationships
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.event.LoggingReceive
+import ztis.UserOrigin
+import ztis.cassandra.SparkCassandraClient
+import ztis.relationships.RelationshipFetcherActor.{FetchRelationshipsTwitter, FetchRelationshipsWykop}
+import ztis.user_video_service.UserServiceActor.{CreateRelationshipsToTwitterUser, CreateRelationshipsToWykopUser, ToTwitterUserRelationshipsCreated, ToWykopUserRelationshipsCreated}
+import ztis.wykop.WykopAPI
+
+import scala.concurrent.Future
+
+object RelationshipFetcherActor {
+
+  case class FetchRelationshipsTwitter(id: Long)
+
+  case class FetchRelationshipsWykop(id: String)
```

Comment:
You still need to use different APIs, create different types of nodes (different properties) so yes I have considered it but we would lose some information about different users.